.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint-staged
```

Comment:
I was wondering if we could just `npx lint-staged` and leave the alias in package.json? but I guess the alias is there as people might see value in running `npm run lint-staged`?

Reply:
Completely right! i hadn't thought about it. I will change it to `npx lint-staged` as I doubt anyone would ever run `npm run lint-staged`